src/components/Map.js

```diff
@@ -115,13 +115,13 @@ function Map() {
             </button>
           </Marker>
         ))}
-        {selectedCourt ? (
+        {court ? (
```

Comment:
```suggestion
        {court && (
        ...
        )}
```
Think you could use a && statement instead of a ternary-operator when returning `null` anyway.